package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "oc-template-react",
+  "version": "0.0.4",
+  "description": "OC-template-react",
+  "main": "src",
+  "scripts": {
+    "precommit": "npm run test",
+    "lint": "eslint --ignore-pattern '/__mocks__/' .",
+    "test": "jest"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/opentable/oc-template-react.git"
+  },
+  "keywords": [
+    "oc-template",
+    "opencomponent",
+    "oc"
+  ],
+  "author": "Nick Balestra <nick@balestra.ch>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/opentable/oc-template-react/issues"
+  },
+  "homepage": "https://github.com/opentable/oc-template-react#readme",
+  "eslintConfig": {
+    "extends": "opentable",
+    "globals": {
+      "describe": true,
+      "test": true,
+      "it": true,
+      "jest": true,
+      "expect": true,
+      "jasmine": true
+    }
+  },
+  "devDependencies": {
+    "cross-spawn": "^5.1.0",
+    "eslint": "^3.14.0",
+    "eslint-config-opentable": "^6.0.0",
+    "eslint-plugin-import": "^1.16.0",
+    "fs-extra": "^2.1.2",
+    "husky": "^0.13.1",
+    "jest": "^18.1.0",
+    "jsdom": "^9.12.0"
+  },
+  "dependencies": {
+    "babel-core": "^6.23.1",
+    "babel-loader": "^6.3.2",
+    "babel-preset-babili": "0.0.11",
+    "babel-preset-es2015": "^6.22.0",
+    "babel-preset-react": "^6.23.0",
+    "babili-webpack-plugin": "0.0.10",
+    "fs-extra": "^2.0.0",
+    "memory-fs": "^0.4.1",
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
+    "uuid": "^3.0.1",
+    "webpack": "^2.2.1"
```

Comment:
all of these should be dev dependencies right? If this is gonna be included on server-side node clients, these modules should kind of be dependency free?